src/EPR.Calculator.API/Builder/Summary/Common/CalcResultSummaryUtil.cs

```diff
@@ -818,4 +796,25 @@ public static decimal GetCommsCostHeaderWithBadDebtFor2bTitle(CalcResult calcRes
         var commsCostHeaderBadDebtProvision = GetCommsCostHeaderBadDebtProvisionFor2bTitle(calcResult);
         return commsCostHeaderWithoutBadDebt + commsCostHeaderBadDebtProvision;
     }
+
+    public static decimal GetCountryOnePlusFourApportionment(CalcResult calcResult, Countries country)
+    {
+        var onePlusFourApportionment = calcResult.CalcResultOnePlusFourApportionment
+            .CalcResultOnePlusFourApportionmentDetails
+            .Single(x => x.Name == CalcResultCommsCostBuilder.OnePlusFourApportionment);
+
+        switch (country)
+        {
```

Comment:
Default case and return 0

Reply:
Thank you Uday, I have fixed the code now